contexts/UserContext.tsx

```diff
@@ -35,7 +36,8 @@ export interface UserContextState {
 export const defaultUserContextState: UserContextState = {
   connectedWallet: undefined,
   account: undefined,
-  address: "",
+  address: undefined,
```

Comment:
I've opted to avoid using the empty string for the address as it leads to confusion about when the address really exists or not. We now always have an `undefined` address when it does not exist.